### ISSUE TITLE
Leader board

### ISSUE THREAD

**Issue by TurboUrbo** [decentfund/kyodo#185]

**Comment by TurboUrbo** [comment]
@igorline  @romochka My proposal of Leader board screen
A member position could be marked by color in the leader board,  so a member could see his/her position among other members.
![](https://i.imgur.com/8BMx042.png)

**Comment by igorline** [comment]
@TurboUrbo would be great to add approx. amount of tokens to receive and it's value in eth and eur

**Comment by TurboUrbo** [comment]
@igorline smth like this?
![](https://i.imgur.com/OfOPW8l.png)

**Comment by igorline** [comment]
@TurboUrbo yep, I think so. But maybe better to merge those columns in one `approx. value` column. And we need to have same column in absolute leader board

**Comment by TurboUrbo** [comment]
@igorline How can we merge Eth and euro? 
"And we need to have same column in absolute leader board" - not sure. Why?

**Comment by TurboUrbo** [comment]
We can have - Absolute leader - Igor - 680 points in current period

**Comment by igorline** [comment]
I'll buidl it and show you what do I mean!

**Comment by TurboUrbo** [comment]
cool!

**Comment by romochka** [comment]
![image](https://user-images.githubusercontent.com/25977424/48269104-66965180-e43f-11e8-9f50-109b3d41b632.png)
leaderboard, or just a page to see how much points were earned by members in domains

**Comment by romochka** [comment]
![image](https://user-images.githubusercontent.com/25977424/48300364-68feb700-e4e4-11e8-9bb1-0abbd375ed3f.png)

**Comment by igorline** [comment]
Hey, @romochka! Looks good to me!
Why there are some values emphasized in the columns?
Can we also make an option to see points earned within specific domain?

**Comment by igorline** [comment]
Converting to epic and moving to latest milestone.
Will add sub issues along

**Comment by igorline** [comment]
@romochka can you, please, help with sorting arrows for domains?

**Comment by romochka** [comment]
![ldrbrd-top](https://user-images.githubusercontent.com/25977424/49217727-aad99980-f3ce-11e8-9d3b-7165dfdfd5a7.png)
![ldrbrd-social](https://user-images.githubusercontent.com/25977424/49217736-add48a00-f3ce-11e8-8f89-4aeda68dc9dd.png)
